tests/pkg_cryptoauthlib_internal-tests/Makefile

```diff
@@ -11,5 +12,6 @@ BOARD_BLACKLIST := stk3200 stk3600 stk3700
 CFLAGS += -DDO_NOT_TEST_CERT
 USEPKG += cryptoauthlib
 USEMODULE += cryptoauthlib_test
+USEMODULE += embunit
```

Comment:
Needed?

Reply:
About the overhead: The library now comes with an additional API to allow for the use of multiple devices in parallel. They added a bunch of test cases for that API and restructured the tests in general, resulting in more code.
I'll add that to the description.

Reply:
And no, apparently not needed. I'll remove that.